internal/tiltfile/k8s.go

```diff
@@ -800,35 +802,54 @@ func (f portForward) Hash() (uint32, error) {
 func intToPortForward(i starlark.Int) (portForward, error) {
 	n, ok := i.Int64()
 	if !ok {
-		return portForward{}, fmt.Errorf("portForward value %v is not representable as an int64", i)
+		return portForward{}, fmt.Errorf("portForward port value %v is not representable as an int64", i)
 	}
 	if n < 0 || n > 65535 {
-		return portForward{}, fmt.Errorf("portForward value %v is not in the range for a port [0-65535]", n)
+		return portForward{}, fmt.Errorf("portForward port value %v is not in the valid range [0-65535]", n)
 	}
 	return portForward{local: int(n)}, nil
 }
 
+const ipReStr = `^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$`
+const hostnameReStr = `^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-]*[A-Za-z0-9])$`
```

Comment:
where did these regexps come from?

Reply:
Not sure what exactly you're asking. These are pretty standard regexps for IPs and hostnames, not sure if it's possible to pinpoint their exact source, cause they are pretty standard, based on relevant RFCs. Here are examples of where they can be encountered online:
IP: http://ipregex.com
Hostname: https://www.regextester.com/23

I've been using them for a while and I don't remember where I took them from, but again, they are pretty standard, so probably shouldn't matter.